examples/transient_adiabatic_mg/c5g7-td-2d-mg.py

```diff
@@ -0,0 +1,142 @@
+import numpy as np
+import os
+import openmc
+import openmc.mgxs
+import openmc.kinetics as kinetics
+
+from geometry_2d_mg import materials, surfaces, universes, cells, lattices, geometry, mgxs_lib_file
+from mgxs_lib import mgxs_data
+materials_file = openmc.Materials(geometry.get_all_materials().values())
+os.environ['OPENMC_MG_CROSS_SECTIONS'] = 'mgxs.h5'
+
+###############################################################################
+# Define problem settings
+
+settings_file = openmc.Settings()
+settings_file.batches = 100
+settings_file.inactive = 40
+settings_file.particles = 1000
+settings_file.output = {'tallies': False}
+
+# Create an initial uniform spatial source distribution over fissionable zones
+source_bounds  = [-32.13, -10.71, -64.26, 10.71,  32.13,  64.26]
+uniform_dist = openmc.stats.Box(source_bounds[:3], source_bounds[3:], only_fissionable=True)
+settings_file.source = openmc.source.Source(space=uniform_dist)
+
+sourcepoint = dict()
+sourcepoint['batches'] = [settings_file.batches]
+sourcepoint['separate'] = True
+sourcepoint['write'] = True
+settings_file.sourcepoint = sourcepoint
+
+# For source convergence checks, add a mesh that can be used to calculate the
+# Shannon entropy
+entropy_mesh = openmc.RegularMesh()
+entropy_mesh.lower_left  = source_bounds[:3]
+entropy_mesh.upper_right = source_bounds[3:]
+entropy_mesh.dimension = [4,4,1]
+settings_file.entropy_mesh = entropy_mesh
+
+###############################################################################
+# Define transient problem parameters
+
+# Create pin cell mesh
+full_pin_cell_mesh = openmc.RegularMesh()
+full_pin_cell_mesh.type = 'regular'
+full_pin_cell_mesh.dimension = [51,51,1]
+full_pin_cell_mesh.lower_left  = [-32.13, -32.13, -64.26]
+full_pin_cell_mesh.width = [1.26,  1.26,  128.52]
+
+# Create assembly mesh
+full_assembly_mesh = openmc.RegularMesh()
+full_assembly_mesh.type = 'regular'
+full_assembly_mesh.dimension = [3,3,1]
+full_assembly_mesh.lower_left  = [-32.13, -32.13, -64.26]
+full_assembly_mesh.width = [ 21.42,  21.42,  128.52]
+
+# Instantiate an EnergyGroups object for the diffuion coefficients
+fine_groups = openmc.mgxs.EnergyGroups()
+fine_groups.group_edges = [0., 0.13, 0.63, 4.1, 55.6, 9.2e3, 1.36e6, 1.0e7]
+
+# Instantiate an EnergyGroups object for the transient solve
+energy_groups = openmc.mgxs.EnergyGroups()
+energy_groups.group_edges = [0., 0.13, 0.63, 4.1, 55.6, 9.2e3, 1.36e6, 1.0e7]
```

Comment:
`energy_groups.group_edges = fine_groups.group_edges`

Reply:
I think this could go either way. It's an example input, so it doesn't really make a difference if you write out the full array again or not. I tend to like laying out the whole thing for the user so they don't get confused and think that energy_groups has to equal fine_groups.